### PR TITLE
Validations refactor

### PR DESCRIPTION
**Pull request by jmendeza**
https://github.com/craftersoftware/craftercms/issues/470

### CODE REVIEW

File: src/main/java/org/craftercms/studio/model/rest/content/UnlockItemByPathRequest.java [hunk 1]
```diff
@@ -20,16 +20,19 @@
...
 import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Size;
 
 import static org.craftercms.commons.validation.annotations.param.EsapiValidationType.HTTPURI;
 import static org.craftercms.commons.validation.annotations.param.EsapiValidationType.SITE_ID;
 
 public class UnlockItemByPathRequest {
 
     @NotEmpty
+    @Size(max = 4)
```
Comment (sumerjabri) [inline src/main/java/org/craftercms/studio/model/rest/content/UnlockItemByPathRequest.java]: What does this limit?

File: src/main/java/org/craftercms/studio/model/rest/content/UnlockItemByPathRequest.java [hunk 2]
```diff
@@ -20,16 +20,19 @@
...
     @EsapiValidatedParam(type = SITE_ID)
     private String siteId;
     @NotEmpty
+    @Size(min = 2)
```
Comment (sumerjabri) [inline src/main/java/org/craftercms/studio/model/rest/content/UnlockItemByPathRequest.java]: Why 2?

File: src/main/resources/crafter/commons/validation/errors_es.properties [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (C) 2007-2022 Crafter Software Corporation. All Rights Reserved.
```
Comment (sumerjabri) [inline src/main/resources/crafter/commons/validation/errors_es.properties]: What's this file about?

**Review by sumerjabri — CHANGES_REQUESTED** [review]

File: src/main/resources/crafter/commons/validation/errors_es.properties (hunk 3 above)
Comment (jmendeza) [reply src/main/resources/crafter/commons/validation/errors_es.properties]: This was a local test, I removed it

File: src/main/java/org/craftercms/studio/model/rest/content/UnlockItemByPathRequest.java (hunk 1 above)
Comment (jmendeza) [reply src/main/java/org/craftercms/studio/model/rest/content/UnlockItemByPathRequest.java]: It was a test. I cleaned it up.

File: src/main/java/org/craftercms/studio/model/rest/content/UnlockItemByPathRequest.java (hunk 2 above)
Comment (jmendeza) [reply src/main/java/org/craftercms/studio/model/rest/content/UnlockItemByPathRequest.java]: It was a test. I cleaned it up.

**Comment by jmendeza** [comment]
This PR depends on https://github.com/craftercms/engine/pull/612

**Review by sumerjabri — APPROVED** [review]

**Review by sumerjabri — APPROVED** [review]

**Pull request merged**